### PR TITLE
COMRS-35 | Add patient's phone number just bellow 'Age' field

### PR DESCRIPTION
**Pull request by prashantsable1991**

### CODE REVIEW

File: openmrs/apps/registration/app.json [hunk 1]
```diff
@@ -109,7 +109,12 @@
                     "label": "REGISTRATION_SEARCH_ADDRESS_LABEL_KEY",
                     "placeholder": "REGISTRATION_SEARCH_ADDRESS_PLACEHOLDER_KEY",
                     "field": "address1"
-                }
+                }, 
+                "customAttributes": {
+                        "label": "REGISTRATION_SEARCH_PHONE_NUMBER",
```
Comment (lauravignoli) [inline openmrs/apps/registration/app.json]: @prashantsable1991 Please fix indenting on this file :)

Comment (prashantsable1991) [reply openmrs/apps/registration/app.json]: Added indentations please review once.

File: openmrs/apps/registration/app.json [hunk 2]
```diff
@@ -1,120 +1,136 @@
 {
-        "id": "bahmni.registration",
```
Comment (lauravignoli) [inline openmrs/apps/registration/app.json]: @prashantsable1991 There's a lot of formatting changes here. Please revert and only commit the code changes you made.

File: openmrs/apps/registration/app.json [hunk 3]
```diff
@@ -1,120 +1,136 @@
...
+            "middleName": {
+                "pattern": "*",
+                "errorMessage": "Should contain characters middle name"
+            },
+            "address1": {
+                "pattern": "*",
+                "errorMessage": "Should contain 3 characters"
+            }
+        },
+        "patientSearch": {
```
Comment (lauravignoli) [inline openmrs/apps/registration/app.json]: @prashantsable1991 This was updated to an old configuration, please revert.

File: openmrs/apps/registration/app.json (hunk 1 above)
Comment (prashantsable1991) [reply openmrs/apps/registration/app.json]: > @prashantsable1991 Please fix indenting on this file :)

@lauravignoli As per your suggestion apply indentation , I applied indentation on whole file. I think it is good practices to do indentation on whole file cause in the end we should look to whole file. 
Going ahead we should follow standard indentation cause every individual developers 
thinking is different for indentation.

**Comment by prashantsable1991** [comment]
Thaks for adding changes.

**Pull request merged**